webserver.md

```diff
@@ -0,0 +1,280 @@
+{::options parse_block_html="true" /}
+
+<div class="row">
+
+<div class="large-12 columns">
+# Build Your Own Internet
+
+Most web applications are served from remote machines in data centers miles and miles away from the devices that are using them. Through the joys of high-speed internet connections that distance can seem trivial and unnoticeable.
+
+Because the Tessel 2 runs a full Linux operating systerm, called [OpenWRT](https://openwrt.org), it can run a server and deliver web applications to locally connected devices, quicker than those remote machines due to the immediate distance between the server and devices.
```

Comment:
Yea I'll work on that phrasing. And it is quicker. Francis did some benchmarking of the Tessel receiving 100 concurrent requests, which resulted in approximately 42.562ms per request. The same request from a browser is ~348ms.